deps/chakrashim/core/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+## Contributing Code
```

Comment:
Thats right. The sources in `core` folder are from [Microsoft/ChakraCore](https://github.com/Microsoft/Chakracore/tree/release/1.1). It is just easier to update the chakracore sources in node deps when they are aligned with their origin in github counterpart.